packages/testing/src/mockRequests.ts

```diff
@@ -61,20 +73,32 @@ export type DataFunction = (
     req,
     ctx,
   }: {
-    req: GraphQLMockedRequest
-    ctx: GraphQLMockedContext<Record<string, any>>
+    req: GraphQLRequest<any>
+    ctx: GraphQLContext<Record<string, any>>
   }
 ) => Record<string, unknown>
 
+// These should get exported from MSW
+type ResponseFunction<BodyType = any> = (
+  ...transformers: ResponseTransformer<BodyType>[]
+) => MockedResponse<BodyType>
+
+type ResponseEnhancers = {
+  once: ResponseFunction<any>
+  networkError: (message: string) => void
+}
+type ResponseEnhancer = keyof ResponseEnhancers
```

Comment:
If the general idea of this PR is good, I can see about exporting these types from `msw` in an upcoming release and dropping these.